src/visualization/StaticGraph/main.cc

```diff
@@ -3,6 +3,17 @@
 #include <SFML/Window.hpp>
 #include <SFML/Graphics/RectangleShape.hpp>
 
+vector<string> giveKmers(vector<Node>& nodeList) {
```

Comment:
should rather be called "getKmers" as is customary. It otherwise sounds as if the kmers are input